portafly/src/i18n/translations/en-us.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
This would be much more readable and maintainable if we use YAML.

Reply:
True, I chose the JSON because it can be imported by JS "natively" but there must be a loader or something for YAML

Reply:
Would be so much better with yml! let's see if we can find a loader/plugin

Reply:
Turns out is not so simple... since we're using CRA, we can't use a webpack-loader or something. We would need to add more libraries (`js-yaml`+`fs`) to read and parse the yaml.. or eject.. or jail-break it using something like `react-app-rewired`..

Another solution could be convert it from YAML to JSON to be consumed by the app, not ideal either.